src/test/java/seedu/bookmark/logic/commands/SortCommandTest.java

```diff
@@ -0,0 +1,86 @@
+package seedu.bookmark.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.bookmark.commons.core.Messages.MESSAGE_BOOKS_SORTED;
+import static seedu.bookmark.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.bookmark.testutil.TypicalBooks.CRIME_AND_PUNISHMENT;
+import static seedu.bookmark.testutil.TypicalBooks.ENDERS_GAME;
+import static seedu.bookmark.testutil.TypicalBooks.HARRY_POTTER;
+import static seedu.bookmark.testutil.TypicalBooks.LORD_OF_THE_FLIES;
+import static seedu.bookmark.testutil.TypicalBooks.ON_THE_ROAD;
+import static seedu.bookmark.testutil.TypicalBooks.THE_HUNGER_GAMES;
+import static seedu.bookmark.testutil.TypicalBooks.TO_KILL_A_MOCKINGBIRD;
+import static seedu.bookmark.testutil.TypicalBooks.getTypicalLibrary;
+
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.bookmark.model.Model;
+import seedu.bookmark.model.ModelManager;
+import seedu.bookmark.model.UserPrefs;
+import seedu.bookmark.model.book.comparators.BookGenreComparator;
+import seedu.bookmark.model.book.comparators.BookNameComparator;
+import seedu.bookmark.model.book.comparators.BookPagesReadComparator;
+
+/**
+ * Contains integration tests (interaction with the Model) for {@code SortCommand}.
+ */
+public class SortCommandTest {
+    private Model model = new ModelManager(getTypicalLibrary(), new UserPrefs());
+    private Model expectedModel = new ModelManager(getTypicalLibrary(), new UserPrefs());
+
+    @Test
+    public void execute_sortName() {
```

Comment:
Perhaps the test cases could follow the format `whatIsBeingTested_descriptionOfTestInputs_expectedOutcome` for consistency with the recommended convention
For e.g. `execute_sortByName_success()` for this method.

Reply:
Will update the test case names. Thanks for the comprehensive review, Wai Kye :)